### DISCUSSION TITLE
Better Sorting

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [community/community#128686] [Other Feature Feedback, Questions, & Ideas]
It would be nice if there were a way to tag repos in lists (or something like that) to make it easier to filter things within the lists.

**Comment by TheGiraffe3** [comment]
Does starring them work?

**Comment by Nehal-Mane** [comment]
While GitHub does not offer a direct feature to tag repositories in lists for easy filtering within the platform itself, there are several strategies and tools you can use to achieve similar

**Comment by Nehal-Mane** [comment]
- Using Labels and Search Filters
- GitHub CLI for Filtering Repositories
- Third-Party Tools and Integrations
- Custom Scripts and Automation

**Comment by WaterSwallow** [comment]
As of now, GitHub does not directly support tagging repositories for personal categorization or sorting within a list. It's a feature that could definitely enhance the organization of repositories for users.

However, you can use the "stars" feature as a workaround. Starring a repository makes it easy to return to the project later. You could "star" the repositories you want to list and then view your starred repositories on your profile `https://github.com/username?tab=stars`.

Another action you might consider is to create a README file in your own repository, where you can list and categorize/repos interesting to you.

Please note that these are just workarounds. Your suggestion could be an excellent added feature for GitHub to consider, and I would recommend raising a feature request in the GitHub community forums.

**Reply by TheGiraffe3** [reply]
Aren’t these the GitHub community forums?
